### PR TITLE
Test env

### PR DESCRIPTION
**Pull request by floalex**
Made following changes to handle env file edge cases in https://github.com/layr-team/Layr/issues/45:
```
// create an empty env file if not exist
fs.closeSync(fs.openSync('./.env', 'w'));

// if one of these keys are missing, generate the necessary keys in env file
if (this.noStellarAccount() || this.noPrivateKey()) {
      if (this.noStellarAccount()) {
        let stellarKeyPair = stellar.generateKeys()

        fileUtils.generateEnvFile({
          'STELLAR_ACCOUNT_ID': stellarKeyPair.publicKey(),
          'STELLAR_SECRET': stellarKeyPair.secret()
        })
      // if stellar account detected but no private secret key
      } else if (this.noPrivateKey() && !this.noStellarAccount()) {
        fileUtils.generateEnvFile();
      }
    }
```

### CODE REVIEW

File: utils/file.js [hunk 1]
```diff
@@ -37,7 +35,7 @@ exports.fileSystem = (function(){
     }
 
     if (envVarsToWrite !== ''){
-      fileSystem.writeFileSync('./.env', envVarsToWrite)
+      fileSystem.appendFileSync('./.env', envVarsToWrite)
```
Comment (floalex) [inline utils/file.js]: Use `appendFileSync` here without rewriting the whole file

**Comment by WilfredTA** [comment]
You can change it to this:

```
// create an empty env file if not exist
fs.closeSync(fs.openSync('./.env', 'w'));

// if one of these keys are missing, generate the necessary keys in env file
if (this.noStellarAccount()) {
      if (this.noStellarAccount()) {
        let stellarKeyPair = stellar.generateKeys()

        fileUtils.generateEnvFile({
          'STELLAR_ACCOUNT_ID': stellarKeyPair.publicKey(),
          'STELLAR_SECRET': stellarKeyPair.secret()
        })
      // if stellar account detected but no private secret key
      } else if (this.noPrivateKey()) {
        fileUtils.generateEnvFile();
      }
    }
```

And it will work the same

**Review by WilfredTA — APPROVED** [review]
